client/src/pages/Home/FooterLogo/FooterLogo.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from "react";
+import "./FooterLogo.css";
+
+export function FooterLogo() {
+	const [isShown, setIsShown] = useState(false);
+	return (
+		<div id="footerlogo">
+			<div className="footerlogo_container">
+				<div className="footer-logo-img_container">
+					<img
+						className="footer-logo-img"
+						src="https://i.postimg.cc/VkdMYP4S/Logo.jpg"
+						alt="logo"
+					/>
+				</div>
+				<div className="fotter-button_container">
+					<div>
+						<p>Do want to be part of us?</p>
```

Comment:
please add `you`: 
`Do you want to be part of us?`

Reply:
Oh, thank you, I added it.